src/Nethermind/Nethermind.Cli/NodeManager.cs

```diff
@@ -119,14 +115,13 @@ public async Task<JsValue> PostJint(string method, params object[] parameters)
             return returnValue;
         }
 
-        public async Task<string> Post(string method, params object[] parameters)
+        public async Task<string?> Post(string method, params object[] parameters)
         {
             return await Post<string>(method, parameters);
         }
 
-        public async Task<T> Post<T>(string method, params object[] parameters)
+        public async Task<T?> Post<T>(string method, params object[] parameters)
```

Comment:
This isn't possible in C# 8, and is the reason I bumped to C# 9.  This method works with either a value type or a reference type and so is kinda nullable sometimes.  By marking the return type as nullable like this C# will do the right thing in each case and make the return type non-nullable for value types and nullable for reference types.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/unconstrained-type-parameter-annotations

Reply:
yes, there has been no C# 9 support in Rider IDE until two weeks ago and then we immediately move pruning_uat to 5.0
we are in the transition, just need patience, we will bump everything to C# 9 when pruning _uat is merged

this thing here was a serious problem before and it is what mostly stopped me from fixing nullables everywhere